README.md

```diff
@@ -355,12 +371,6 @@ The `th2_check1_active_tasks_number` metric separate rules with label `rule_type
 + Migrated `sailfish-utils` version from `3.12.2` to `3.14.0`
     + sailfish updated to 3.3.54
 
-### 3.10.0
```

Comment:
Maybe save this records for history and add information about merging 3.10.0 for 4.4.0 version?